src/common/models/project.go

```diff
@@ -211,5 +212,6 @@ type ProjectSummary struct {
 	GuestCount        int64 `json:"guest_count"`
 	LimitedGuestCount int64 `json:"limited_guest_count"`
 
-	Quota *QuotaSummary `json:"quota,omitempty"`
+	Quota    *QuotaSummary   `json:"quota,omitempty"`
+	Registry *model.Registry `json:"registry"`
```

Comment:
Would it expose too much information in the response?
I think all it needs is endpoint and may be ID?

Reply:
The `URL` itself isn't enough to show the registry information as users can create two registries with the same `URL` but different `Credential`.  So the `ID`, `Name`, `Description`, `Type` and `URL` can describe the overall of a registry.
If so, hiding the other attributes(`Insecure`, `Status`, `CreationTime`, `UpdateTime`) isn't very meaningful?